schemas/when_rule/definitions.json

```diff
@@ -24,6 +24,10 @@
     },
     "minItems": 1
   },
+  "section": {
+    "type": "string",
+    "enum": ["end"]
```

Comment:
I think 'summary' would be a more specific value here. `end` is open to interpretation as it could be the last block, or the summary

Reply:
Section end can take you to a section summary ,the last block, the summary or the hub. It's being used as a routing_path end flag. Because blocks don't have to have section summaries I would argue that we can't be specific.

Reply:
Is this definition used anywhere?

Reply:
Ah crap, good spot, I put it in the when, then released that was completely in the wrong place, moved it out to the common definitions and clearly forgot to delete it